components/Player/PlayerThumbnail/PlayerThumbnail.tsx

```diff
@@ -48,6 +48,7 @@ const PlayerThumbnail: React.FC<IPlayerThumbnailProps> = ({
   return (
     srcUrl && (
       <div
+        title={`Thumbnail for "${title}".`}
```

Comment:
We don't need a title here. The alt text on the image is all we need for the thumbnail.